components/discord/sources/new-message/test-event.mjs

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable pipedream/required-properties-description */
+/* eslint-disable pipedream/required-properties-version */
+/* eslint-disable pipedream/required-properties-name */
+/* eslint-disable pipedream/required-properties-key */
+export default {
+  "channelId": "1064400555309802049",
+  "guildId": "1064400554651292156",
+  "id": "1127815108093289054",
+  "createdTimestamp": 1697448563933,
+  "type": "CUSTOM",
+  "system": true,
+  "content": "Hello, Universe",
```

Comment:
```suggestion
  "content": "Hello, universe",
```

Reply:
Changed "Universe" to lowercase to follow sentence case rule from Google's Developer docs style guide.